doc/usecases/snippet_sharing/UC_edit_snippet.md

```diff
@@ -0,0 +1,19 @@
+# Use Case: Edit configuration snippet
+
+## Summary
+
+Title: Edit configuration snippet
+Scope: Entry Management
+Level: User Goal
+Actors: Authenticated user (snippet owner), Moderator
+Brief: Users can edit their own database entries. Moderators can edit all configuration snippets in the database, also the ones of other users.

```

Comment:
Is it possible to do OAUTH on the client side? Maybe its possible to store some information at github and use them for registration/login?

> No idea if GitHub does allow something like this, but that is definitely a total-minor on my list.

Seems [to be available](https://developer.github.com/v3/issues/#create-an-issue) and could be really nice for manual testing of plugins. But sending an e-mail or simply logging errors is okay, too.